docs/source/writing-daml.rst

```diff
@@ -4,6 +4,14 @@
 Writing Daml
 =============
 
+Daml is a smart contract language designed to build composable applications on the `Daml Ledger Model <https://docs.daml.com/concepts/ledger-model/index.html#da-ledgers>`_.
+
+The Writing Daml section will teach you how to write Daml applications that run on any Daml Ledger implementation, including key language features, how they relate to the Daml Ledger Model, and how to use Daml’s developer tools to write, test, compile, package and ship your application. It also covers the structure of a Daml Ledger as it pertains to designing your application.
```

Comment:
I see you are both using and not using the Oxford comma here. Is this by choice? Honest question, I know you can be flexible and use it when it actually helps to disambiguate the meaning and skip when it's not, but I'd prefer asking to the expert rather than guessing. 🙂

Reply:
It's for flow and clarity but now that you raise this, I think using a bulleted list is probably better for this specific context!